app.json

```diff
@@ -1,20 +1,32 @@
 {
   "name": "ah-frontend-athena",
   "scripts": {},
-  "env": {},
+  "env": {
+    "REACT_APP_GOOGLE_SECRET_ID": {
+      "required": true
+    },
+    "REACT_APP_GOOGLE_CLIENT_ID": {
+      "required": true
+    },
+    "REACT_APP_FACEBOOK_APP_ID": {
+      "required": true
+    },
+    "REACT_APP_API_URL_BASE": {
+      "required": true
+    }
+  },
   "formation": {
     "web": {
       "quantity": 1
     }
   },
   "addons": [],
-  "buildpacks": [
-    {
+  "buildpacks": [{
       "url": "heroku/nodejs"
     },
     {
       "url": "https://github.com/mars/create-react-app-buildpack"
     }
   ],
   "stack": "heroku-18"
-}
+}
```

Comment:
add new lines at the end of files.

Reply:
I had a problem with the extension that was helping me do this (blank line). it excluded `.json` files and other types. Enabling it on these files didn't help. I solved it by changing vs code settings to add a new line at the end of a file.